packages/story-editor/src/components/library/panes/pageTemplates/pageTemplatesPane.js

```diff
@@ -178,12 +185,14 @@ function PageTemplatesPane(props) {
     return () => clearTimeout(timeout);
   }, [highlightedTemplate]);
 
-  const options = [
-    {
+  const options = [];
```

Comment:
Let's hide `DropDownWrapper` below if `options.length <= 1`

The dropdown is not useful when there is only one option or no option at all.

Reply:
So just don't show the dropdown at all? Or show a title?

Reply:
<img width="363" alt="Screenshot 2022-01-17 at 15 12 09" src="https://user-images.githubusercontent.com/237508/149794971-286c2c59-5b61-4a94-a27c-77e51a82b641.png">
Spacing on this looks off.

Reply:
I see.

Let's do it this way then:

1. Only hide the `<Select>`, but not the `DropDownWrapper`
1. Remove height from `DropDownWrapper`. It serves no purpose anyway. Can actually also remove the `text-align`

So essentially:

```js
const DropDownWrapper = styled.div`
  margin: 28px 16px 17px;
`;

{/* Still rendering DropDownWrapper to keep spacing nice and tidy */}
<DropDownWrapper>
  {options.length > 1 && (
    <Select
      options={options}
      selectedValue={showDefaultTemplates ? DEFAULT : SAVED}
      onMenuItemClick={handleToggle}
      aria-label={__('Select templates type', 'web-stories')}
    />
  )}
</DropDownWrapper>
```

With the result looking like this:

![Screenshot 2022-01-17 at 23 18 14](https://user-images.githubusercontent.com/841956/149843099-bd597012-e25a-4830-bcc7-8528479dfc2b.png)

That'll do for now

I'll ping UX if needed in the future.

Reply:
Fixed.